pkg/solana/logpoller/models.go

```diff
@@ -0,0 +1,40 @@
+package logpoller
+
+import (
+	"time"
+
+	"github.com/gagliardetto/solana-go"
+	"github.com/lib/pq"
+)
+
+type Filter struct {
+	ID            int64
+	ChainId       string
+	Name          string
+	Address       solana.PublicKey
+	EventName     string
+	EventSig      []byte
+	StartingBlock int64
+	EventIDL      string
+	SubKeyPaths   []string
+	Retention     int64
```

Comment:
Why not `time.Duration`? That's what we use for the Retention field of `logpoller.Filter`